hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/util/S3StorageType.java

```diff
@@ -52,4 +52,12 @@ public static S3StorageType getDefault() {
     return STANDARD;
   }
 
+  public static S3StorageType fromReplicationType(
+      ReplicationType replicationType) {
+    if (replicationType == ReplicationType.STAND_ALONE) {
+      return S3StorageType.REDUCED_REDUNDANCY;
```

Comment:
just a question, I think this is before even this patch.
Previously we use STAND_ALONE for replication factor one, now we use RATIS with one and three. So, I think we need to change the code for this right?

Reply:
Sounds reasonable but it requires the change of the protobuf (factor is not available from the message). Seems to be a bigger change but I added it to this patch (sorry, now it's an other commit to review :-P )